packages/ra-core/src/controller/list/useListParams.ts

```diff
@@ -82,7 +82,7 @@ export const useListParams = ({
     perPage = 10,
     resource,
     sort = defaultSort,
-    storeKey = `${resource}.listParams`,
+    storeKey = disableSyncWithLocation ? '' : `${resource}.listParams`,
```

Comment:
Shouldn't the default value be `false` by default when `disableSyncWithLocation === true`?